system/jlib/jstats.cpp

```diff
@@ -985,13 +985,6 @@ IAtom * createActivityScope(unsigned value)
     return createStatsScope(temp);
 }
 
-static IAtom * createSubGraphScope(unsigned value)
```

Comment:
I think it is something that changed, and hasn't been updated.  It looks like createActivityScope above and createEdgeScope below are also unused.